src/condor_starter.V6.1/jic_shadow.cpp

```diff
@@ -2741,178 +2748,181 @@ JICShadow::verifyXferProgressing(int /*timerid*/) {
 int
 JICShadow::transferInputStatus(FileTransfer *ftrans)
 {
-	// An in-progress message? (ftrans is null when we fake completion)
-	if (ftrans) {
-		const FileTransfer::FileTransferInfo &info = ftrans->GetInfo();
-		if (IsDebugCategory(D_ZKM)) {
-			std::string buf;
-			info.dump(buf,"\t");
-			if (info.stats.size()) { formatAd(buf,info.stats,"\t",nullptr,false); }
-			dprintf(D_ZKM /* | (info.in_progress ? 0 : D_BACKTRACE) */, "starter transferInputStatus: %s", buf.c_str());
-		}
-		if (info.in_progress) {
-			// a status ping message. xfer is still making progress!
-			this->file_xfer_last_alive_time = time(nullptr);
-			return 1;
-		}
+	ASSERT(ftrans);
+	const FileTransfer::FileTransferInfo & info = ftrans->GetInfo();
```

Comment:
I think we should take the time here to not have two reference variables to the same object (see line 2779). I know this is due to how the code was originally implemented (checking that we have a FileTransfer object pointer before referencing info)